src/window.rs

```diff
@@ -312,22 +315,28 @@ impl Window {
             _ => true,
         };
 
+        let maximized = window_config.maximized();
+
```

Comment:
Tiny nitpick, I think it would be cleaner to just put `window_config.maximized()` directly into the `with_maximized` call. Should reduce the code size while at the same time making it easier to read since you don't need to look up the variable.